test/e2e/admin/admin-statistics.test.ts

```diff
@@ -282,7 +282,7 @@ class AdminStatisticsIntegrationTest extends IntegrationTest {
       query.text,
       query.values,
     ) as object[];
-    this.expect(result[1]).to.deep.equal(count[1]);
+    this.expect(count[1]).to.deep.equal(result[1]);
```

Comment:
What exactly changes by flipping the ordering? (here and other places)

Reply:
It just helps with debugging so that when the message gives:
expected: A
actual: B
it has the correct ordering